net/dawn/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# This software is licensed under the Public Domain.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dawn
+PKG_SOURCE_DATE:=2020-03-23
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/berlin-open-wireless-lab/DAWN.git
+PKG_SOURCE_VERSION:=f03f55ff92153337972cb9b2d1856b9a364df073
+PKG_MIRROR_HASH:=065ae3c9d04881f68da775f86c4def8cf8e19abce12bac9ac3b4145a66e33e56
+
+PKG_MAINTAINER:=Nick Hainke <vincent@systemli.org>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/dawn
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Decentralized wifi controller
+  URL:=https://github.com/berlin-open-wireless-lab/DAWN.git
+  DEPENDS:=$(DRV_DEPENDS) +libubus +libubox +libblobmsg-json +libuci +libgcrypt +libiwinfo +umdns
+endef
+
+define Package/dawn/description
+  This package implements a decentralized wireless daemon.
+endef
+
+define Package/dawn/install
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/dawn $(1)/usr/sbin/dawn
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/dawn.init $(1)/etc/init.d/dawn
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_CONF) ./files/dawn.config $(1)/etc/config/dawn
+	$(INSTALL_DIR) $(1)/tmp
```

Comment:
Uhhh what?

This will get wiped on boot.

If you need an empty file in /tmp, use touch in the init script.

Reply:
That is true! Someone demanded that the file should be located in `/tmp/`. That would be a huge problem. Thanks a lot! :)